source/_posts/2021-12-11-release-202112.markdown

```diff
@@ -961,6 +961,25 @@ Dataclasses provided by discovery type:
 
 {% enddetails %}
 
+{% details "Hue" %}
```

Comment:
Let's move this up to the breaking change section (it is now in the section for custom integration developers).